tensorrt_llm/bench/benchmark/low_latency.py

```diff
@@ -45,13 +47,23 @@
     default=None,
     help="Path to a serialized TRT-LLM engine.",
 )
+@optgroup.option(
```

Comment:
If we make a new option group for these, we can classify them as mutually exclusive. I don't think we should allow both to be specified.